smoke-tests/src/test/groovy/io/opentelemetry/smoketest/JettySmokeTest.groovy

```diff
@@ -15,7 +15,7 @@ package io.opentelemetry.smoketest
 @AppServer(version = "10.0.0", jdk = "15-openj9")
 class JettySmokeTest extends AppServerTest {
 
-  protected String getTargetImage(String jdk, String serverVersion) {
-    "ghcr.io/open-telemetry/java-test-containers:jetty-${serverVersion}-jdk$jdk-20210223.592806654"
+  protected String getTargetImagePrefix() {
+    "ghcr.io/open-telemetry/java-test-containers:jetty"
```

Comment:
I think we decided here that versioning is not important for Java app container tests - it's probably because if versioning caused a problem, other tests would fail. I think it's ok but don't know if we've documented such a decision anywhere

Reply:
Who decided? When? `AppServerTest` still requests a specific container version.

Reply:
Oh I didn't put them together - I guess XXX is waiting for the other fix? Then no worries.